src/main/java/io/github/debuggyteam/architecture_extensions/DeferredRegistration.java

```diff
@@ -22,7 +22,7 @@
 
 public class DeferredRegistration {
 	private static Multimap<Identifier, Entry> deferrals = MultimapBuilder.hashKeys().arrayListValues(2).build();
-	
+
```

Comment:
lots of extraneous formatting - intellij is deleting whitespace without telling you again

Reply:
I use vscode not intellij.